lib/e164/lookuptable.rb

```diff
@@ -398,13 +348,88 @@ module E164
     "1978" => ["US", "United States"],
     "1979" => ["US", "United States"],
     "1980" => ["US", "United States"],
     "1984" => ["US", "United States"],
     "1985" => ["US", "United States"],
     "1986" => ["US", "United States"],
     "1989" => ["US", "United States"],
+    "1204" => ["CA", "Canada"],
+    "1226" => ["CA", "Canada"],
+    "1236" => ["CA", "Canada"],
+    "1242" => ["BS", "Bahamas"],
+    "1246" => ["BB", "Barbados"],
+    "1249" => ["CA", "Canada"],
+    "1250" => ["CA", "Canada"],
+    "1264" => ["AI", "Anguilla"],
+    "1268" => ["AG", "Antigua and Barbuda"],
+    "1284" => ["VG", "British Virgin Islands"],
+    "1289" => ["CA", "Canada"],
+    "1306" => ["CA", "Canada"],
+    "1340" => ["VI", "Virgin Islands, US"],
+    "1343" => ["CA", "Canada"],
+    "1345" => ["KY", "Cayman Islands"],
+    "1365" => ["CA", "Canada"],
+    "1367" => ["CA", "Canada"],
+    "1403" => ["CA", "Canada"],
+    "1416" => ["CA", "Canada"],
+    "1418" => ["CA", "Canada"],
+    "1431" => ["CA", "Canada"],
+    "1437" => ["CA", "Canada"],
+    "1438" => ["CA", "Canada"],
+    "1441" => ["BM", "Bermuda"],
+    "1450" => ["CA", "Canada"],
+    "1473" => ["GD", "Grenada"],
+    "1506" => ["CA", "Canada"],
+    "1514" => ["CA", "Canada"],
+    "1519" => ["CA", "Canada"],
+    "1548" => ["CA", "Canada"],
+    "1581" => ["CA", "Canada"],
+    "1587" => ["CA", "Canada"],
+    "1600" => ["CA", "Canada"],
+    "1604" => ["CA", "Canada"],
+    "1613" => ["CA", "Canada"],
+    "1622" => ["CA", "Canada"],
+    "1639" => ["CA", "Canada"],
+    "1647" => ["CA", "Canada"],
+    "1649" => ["TC", "Turks and Caicos Islands"],
+    "1664" => ["MS", "Montserrat"],
+    "1670" => ["MP", "Northern Mariana Islands"],
+    "1671" => ["GU", "Guam"],
+    "1672" => ["CA", "Canada"],
+    "1684" => ["AS", "American Samoa"],
+    "1705" => ["CA", "Canada"],
+    "1709" => ["CA", "Canada"],
+    "1721" => ["SX", "Sint Maarten (Dutch)"],
+    "1758" => ["LC", "Saint Lucia"],
+    "1767" => ["DM", "Dominica"],
+    "1778" => ["CA", "Canada"],
+    "1780" => ["CA", "Canada"],
+    "1782" => ["CA", "Canada"],
+    "1784" => ["VC", "Saint Vincent and the Grenadines"],
+    "1787" => ["PR", "Puerto Rico"],
+    "1800" => ["US", "Toll Free"],
+    "1807" => ["CA", "Canada"],
+    "1809" => ["DO", "Dominican Republic"],
+    "1819" => ["CA", "Canada"],
+    "1825" => ["CA", "Canada"],
+    "1829" => ["DO", "Dominican Republic"],
+    "1844" => ["US", "Toll Free"],
+    "1849" => ["DO", "Dominican Republic"],
+    "1855" => ["US", "Toll Free"],
+    "1866" => ["US", "Toll Free"],
+    "1867" => ["CA", "Canada"],
+    "1868" => ["TT", "Trinidad and Tobago"],
+    "1869" => ["KN", "Saint Kitts and Nevis"],
+    "1873" => ["CA", "Canada"],
+    "1876" => ["JM", "Jamaica"],
+    "1877" => ["US", "Toll Free"],
+    "1888" => ["US", "Toll Free"],
+    "1902" => ["CA", "Canada"],
+    "1905" => ["CA", "Canada"],
+    "1939" => ["PR", "Puerto Rico"],
```

Comment:
This one is now a duplicate entry.

Reply:
Old view of code. I specifically installed rubocop to lint duplicate hash values (there are none now).